doc/specs/SCOOBE.md

```diff
@@ -0,0 +1,185 @@
+
+**\*** - By storing the content for SCOOBE externally from the application, the PowerToys team can update/adjust information without being constrained to PowerToys&#39; release cycles. This is critical in the event of errors or miscommunications that would otherwise be difficult, if not impossible, to correct if stored locally to the app and shipped with the version of PowerToys being released.
+
+**\*\*** - It is important to call out the nuances of potential upgrade scenarios here. If a user upgraded from v0.31 to v0.33, the updates displayed would include only what was changed in v0.33, as this scenario only adds a single release&#39;s changes. Should a user have upgraded from v0.29 -> v0.33, the updates displayed would include what was changed in both v0.31 and v0.33 as this scenario involves multiple releases&#39; changes. If the latest version of PowerToys is v0.35 (for example), but the user only updates from v0.31 to v0.33, the displayed information **should not** include information related to v0.35 updates, as those changes have not yet been added on the version of PowerToys the user has installed. Section 5.1.3 gives textual examples of what various scenarios would look like, with section 3.1.2 describing the page&#39;s content layout.
+
+**3.1.2. Page Content**
+
+| **No.** | **Requirement** | **Pri** |
+| --- | --- | --- |
+|1. | The SCOOBE dialog should display the version of PowerToys the user has installed, as shown in figures 5.1.1. | P0 |
+|2. | The SCOOBE dialog information should be grouped into two sections: &quot;New Features &amp; Improvements&quot; and &quot;Bugfixes Highlights&quot;. | P0 |
+|3. | The &quot;New Features &amp; Improvements&quot; section should contain information related to end user functionality that has been added or updated. | P0 |
+|4. | The &quot;New Features &amp; Improvements&quot; section should be subdivided by the relevant utilities updated (i.e. Color Picker, FancyZones, etc.). See figure 5.1.1 and section 5.1.3 for examples. | P1 |
+|5. | The &quot;Bugfixes Highlights&quot; section should contain information related to noteworthy issues/errors that were corrected. | P0 |
+|6. | If there are relevant visuals, they should be included with the information text. See figure 5.1.1. | P1 |
+|7. | For multi-version updates (i.e. v0.29 to v0.35), combine the information from each version&#39;s updates and display them as a single &quot;New Features &amp; Improvements&quot; section and &quot;Bugfixes Highlights&quot; section. See section 5.1.3 for an example. | P0 |
+|8. | The SCOOBE dialog should be scrollable if needed to fit all the content. | P0 |
+|10. | The bottom of the SCOOBE dialog should include a link to the PowerToys releases page for a complete list of versions and their release notes ([Releases · microsoft/PowerToys (github.com)](https://github.com/microsoft/PowerToys/releases)). | P1 |
+
+**3.2. Open Discussion**
+
+- How do we localize content like keystrokes and visuals?
```

Comment:
I would localize keystrokes not until #9482 is  implemented

Reply:
The problem is here is not just how to localize content like keystrokes, the problem is the added complexity to the entire localization process and the fact that SCOOBE needs to implement its own logic to determine what locale should be used, adding another layer of complexity for both development and testing.